index.js

```diff
@@ -70,232 +63,224 @@ module.exports.ProfileHooker = async (settings = { username: "", cookie: "", tim
 
 
 module.exports.PostHooker = async (settings = { url: "", cookie: "", timeout: 50, output: "" }) => {
-    const options = {
-        url: settings.url,
-        cookie: `sessionid=${settings.cookie};`,
-        timeout: settings.timeout ? settings.timeout : 50,
-        output: settings.output || "instahooker-out"
-    }
-
-    const posts = [];
+  const options = {
+        "url": settings.url,
+        "cookie": `sessionid=${settings.cookie};`,
+        "timeout": settings.timeout ? settings.timeout : 50,
+        "output": settings.output || "instahooker-out"
+      }, posts = [],
+      [data] = await Promise.all([fetch(`${baseURL}/p/${options.url.startsWith(baseURL) ? postRegex.exec(options.url)[3] : options.url}/?__a=1`, {headers: {cookie: options.cookie}})
+          .then(res => res.json())]);
 
-    const data = await fetch(`${baseURL}/p/${options.url.startsWith(baseURL) ? postRegex.exec(options.url)[3] : options.url}/?__a=1`, { headers: { cookie: options.cookie } })
-        .then(res => res.json());
 
-    getData(data, posts, "post");
+  getData(data, posts, "post");
 
-    //Check the posts array
-    if (!posts.length && data.graphql.user.is_private)
-        return logger("warn", "The account you want to reach is private and you don't follow it, so media cannot be downloaded!");
+  //Check the posts array
+  if (!(posts.length || !data.graphql.user.is_private))
+    return logger("warn", "The account you want to reach is private and you don't follow it, so media cannot be downloaded!");
 
-    //Show info for download
-    await logger("ready");
-    await getRemoteFile(posts, `./${options.output}/`, options.timeout);
+  //Show info for download
+  await logger("ready");
+  await getRemoteFile(posts, `./${options.output}/`, options.timeout);
 
 }
 
 
 module.exports.StoryHooker = async (settings = { username: "", cookie: "", timeout: 50, output: "", highlight: false }) => {
-
-    const options = {
-        username: settings.username,
-        cookie: `sessionid=${settings.cookie};`,
-        timeout: settings.timeout ? settings.timeout : 50,
-        output: settings.output || "instahookerStory-out",
-        highlight: settings.highlight || false
+  const options = {
+    "username": settings.username,
+    "cookie": `sessionid=${settings.cookie};`,
+    "timeout": settings.timeout ? settings.timeout : 50,
+    "output": settings.output || "instahookerStory-out",
+    "highlight": settings.highlight || false

```

Comment:
Remove double quotes from here too.

Reply:
> 

they have been changed to as name space there is no vulnerability in the removal